src/styles/index.scss

```diff
@@ -1,3 +1,81 @@
+$stopwatch-size: 80vmin;
+$center-size: 5vmin;
+$minutes-hand-color: #0700ff;
+$seconds-hand-color: #2c8000;
+$center-color: #f6a603;
+$minutes-hand-width: 3vmin;
+$minutes-hand-height: 20vmin;
+$seconds-hand-width: 1.5vmin;
+$seconds-hand-heigth: 38vmin;
```

Comment:
There is a typo in the variable name `$seconds-hand-heigth`. It should be `$seconds-hand-height` to maintain consistency and avoid potential issues in the styling.